package.json

```diff
@@ -16,8 +16,11 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^11.0.0",
     "@rollup/plugin-typescript": "^8.0.0",
+    "@smui/form-field": "^6.0.0-beta.15",
```

Comment:
I don't actually use this, but I think we probably will. I can remove it though.